app/controllers/establish_claims_controller.rb

```
@@ -10,10 +10,14 @@ def perform
   def review
     # Future safeguard for when we give managers a show view
     # for a given task
-    task.start! if current_user == task.user
+    task.start! if current_user == task.user && !task.started?
```

Oh I see what happened here. I missed it in your last PR.

You added a hard fail [here](https://github.com/department-of-veterans-affairs/caseflow/blob/954f90700834ace621ce57d7dbfc6c3522bf2d7c/app/models/task.rb#L101). When I originally added the `start!` method, Shane and I had discussed whether start should error on a 2nd call or silently fail, and we concluded it was more elegant for the code to be able to blindly call `start!`. Since start is really only tracking metadata on when the task was first viewed, we were more loose on this one than other transitions. 

But if you feel strongly for hard failing this is fine with me 👍